### PR TITLE
Security fine improvements

### PR DESCRIPTION
**Pull request by Chatauscours**
## About the PR 
- Lets Security Officers and Inspectors issue fines for up to 500 credits without the approval of a head.
- Fines above 500 credits must be approved by one of: Captain, HoP, HoS, NTSC.
- When a fine is approved (or simply issued for fines of 500 and less) sends a PDA message to the person who got fined stating the issuer of the fine, amount fined and reason.

## Why's this needed? 
- Gives secoffs a newish step of escalation between a simple warning and brig time.
- People should know when they get a fine.

## Changelog 

```changelog
(u)Chatauscours
(*)Security officers and Inspectors can now isssue fines for up to 500 credits without further approval.
(+)You will now be alerted by PDA when you recieve a fine.
```

### CODE REVIEW

**Review by pgmzeta — CHANGES_REQUESTED** [review]
I like this idea, but needs some de-dupe. Easy fixes :)

File: code/obj/item/device/pda2/smallprogs.dm [hunk 1]
```diff
@@ -1160,8 +1162,11 @@ Using electronic "Detomatix" SELF-DESTRUCT program is perhaps less simple!<br>
 					p.name = "Official Fine Notification - [ticket_target]"
 					p.info = ticket_text
 					p.icon_state = "paper_caution"
+
+			else if(fine_amount <= 500)
+				message = "Fine request created, awaiting approval from the Captain, Head of Security,  Head of Personnel, Nanotrasen Security Consultant, Security Officer or Inspector."
 			else
-				message = "Fine request created, awaiting approval from the Captain, Head of Security or Head of Personnel."
+				message = "Fine request created, awaiting approval from the Captain, Head of Security, Head of Personnel or Nanotrasen Security Consultant."
```
[the remark this answers is not in the document]
Comment (TobleroneSwordfish) [reply code/obj/item/device/pda2/smallprogs.dm:1169]: The proc is `english_list`

**Comment by Chatauscours** [comment]
Should be all good now!

File: code/datums/datacore.dm [hunk 2]
```diff
@@ -415,7 +416,6 @@
 	var/datum/db_record/bank_record = null
 	var/target_byond_key = null
 	var/issuer_byond_key = null
-	var/approver_byond_key = null
```
Comment (TobleroneSwordfish) [inline code/datums/datacore.dm:418]: What's the reason for removing this? Approver could still be different to issuer

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]
Minor thing: auto-approved fines still show up in the "Unpaid" section

File: code/datums/datacore.dm (hunk 2 above)
Comment (Chatauscours) [reply code/datums/datacore.dm:418]: The var wasn't used anywhere, and the logging uses usr. Didn't think there was a particular need for a var, but putting it back

**Comment by Chatauscours** [comment]
> Minor thing: auto-approved fines still show up in the "Unpaid" section
No offense, but are you sure you had enough money on the account? I couldn't reporduce it on my side

**Review by TobleroneSwordfish — APPROVED** [review]
Oooh yep I didn't have enough in the account, duh!

**Pull request merged**